src/index.ts

```diff
@@ -15,8 +16,30 @@ export async function main(): Promise<void> {
         process.exit(1);
     }
 
-    const qr = new QuestionRenderer(rootQuestion);
-    const answers = await qr.render();
+    let answers = {};
+
+    if(process.argv.slice(2).length > 0){
+      const args = await yargs(process.argv.slice(2)).options({
+        clientType: { type: 'string', demandOption: true, choices:['besu','goquorum'], describe: 'Ethereum client to use.' },
```

Comment:
Can we set the same defaults as the questions please for all options and make only the `clientType` and `privacy` as required?

Reply:
@joshuafernandes  All done! 

As for your previous issue, you need to chuck an extra -- before you list the args:
``
npx quorum-dev-quickstart -- --clientType besu --outputPath ./ --elk true -privacy true --orchestra true
``